roseus_smach/src/state-machine-ros.l

```diff
@@ -27,6 +27,7 @@
    (ros::advertise
     (concatenate string srv-name "/smach/container_structure")
     smach_msgs::SmachContainerStructure 100 t) ;; latch type
+   (unix::usleep (round (* 0.5 1e6)))
```

Comment:
please add comments, why we have to add this sleep along with https://github.com/jsk-ros-pkg/jsk_roseus/pull/695 URL, so that we can fix this in future.

Reply:
I updated.